directives/questionnaire.py

```diff
@@ -303,6 +304,10 @@ def add_feedback(self, node, data, paragraph):
             if value.startswith('regexp:'):
                 compare_regexp = True
                 value = value[7:]
+            if enumeration_of_choices and not isnot:
+                line[0] = enumeration_of_choices[value] + content.strip()
+            else:
+                line[0] = content.strip()
```

Comment:
When the ``!`` symbol is prepended, the numeration should be still added. Otherwise, it is a bit annoying to have one line of feedback without the numeration.